_config/trailing-slash.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
@pack I see what you're trying to do here but we wouldn't want to enforce this config at a module level.  I'll push an alternative fix.